tests/functional/settings_devices.js

```diff
@@ -132,7 +132,7 @@ define([
         .findByCssSelector('.device:nth-child(1) .device-name')
           .getVisibleText()
           .then(function (val) {
-            assert.equal(val, BROWSER_DEVICE_NAME + ' (current)', 'device name is correct');
+            assert.equal(val, BROWSER_DEVICE_NAME, 'device name is correct');
```

Comment:
Can you add a test to ensure the current device container element is shown when expected?